docs/source/configuration/internalproxy.md

```diff
@@ -55,6 +71,11 @@ or use the environment variable:
 RAZZLE_DEV_PROXY_API_PATH= RAZZLE_API_PATH=http://localhost:8081/mysite yarn start
 ```
 
+If you disable the proxy, the frontend Volto javascript process in the browser will directly query 
```

Comment:
I think I removed the section "disabling the proxy".

Reply:
removed the paragraph on disabling